src/api/noises.ts

```
@@ -0,0 +1,47 @@
+  if (!noise) {
+    throw new Error('No file selected')
+  }
+
+  const formData = new FormData()
+  formData.append('title', noise.name.split('.')[0])
+  formData.append('fileType', noise.type) // more accurate than name.split
+  formData.append('file', noise)
+
+  // DEBUG: Log formData properly
+  for (const [key, value] of formData.entries()) {
+    console.log(`${key}:`, value)
```

ESLint

[error] Unexpected console statement.

(no-console)